### PR TITLE
Fix ingest timezone

### PR DESCRIPTION
**Pull request by hexa17**

### CODE REVIEW

**Comment by hexa17** [comment]
I fixed hardcoding, put `tzone` inside providers,

File: datalabframework/spark/filter.py [hunk 1]
```diff
@@ -16,20 +16,27 @@ def filter_by_date(obj, options):
...
     if not start_date and window:
         start_date = end_date - window
 
     # build condition
+    if '_date' in obj.columns:
+        obj = obj.filter(F.to_timestamp('_date') < end_date)
+        obj = obj.filter(F.to_timestamp('_date') >= start_date) if start_date else obj
+
+#     obj = obj.filter(F.to_utc_timestamp(F.to_timestamp(column), tzone) < end_date)
+#     obj = obj.filter(F.to_utc_timestamp(F.to_timestamp(column), tzone) >= start_date) if start_date else obj
```
Comment (natbusa) [inline datalabframework/spark/filter.py]: Please uncomment lines. 
Code must compare UTC timestamps.

Comment (hexa17) [reply datalabframework/spark/filter.py]: OK, I updated :( plz check!

**Comment by natbusa** [comment]
Looks good to me. More speedup optimization can be done on a separate commit.

**Pull request merged**